intro-HPC/ch_running_jobs_with_input_output_data.tex

```diff
@@ -751,10 +756,20 @@ \subsection{Setting per-member VO quota}
         100\% can actually be used.
 \end{enumerate}
 
-\begin{figure}[h]
+\begin{figure}[!htbp]
   \caption{Setting per-member quota.}
   \centering
     \includegraphics[width=\textwidth]{ch6-VO-per-member-storage}
-\end{figure}
+\end{figure}\label{fig:VO-per-member-storage}
+
+\subsection{VO directories}
+\label{subsec:vo-directories}
+
+When you're in a VO, there will be some extra directories available:
+
+\begin{tabular}{|p{\dimexpr 0.35\textwidth-2\tabcolsep}|p{\dimexpr 0.65\textwidth-2\tabcolsep}|} \hline
+\strong{Variable} & \strong{Description} \\ \hline\hline
+\$VSC\_SCRATCH\_VO      & The scratch filesystem for your VO storage, this can have a bigger quota than your personal scratch.\\ \hline
```

Comment:
better: `A directory on the shared scratch filesystem shared by the members of your VO, where additional storage quota can be provided (see relevant section?). You can use this as an alternative to your personal \verb|$VSC_SCRATCH| directory (see relevant section).`

also:

* `$VSC_SCRATCH_VO_USER` (personal directory in VO space, equivalent with `$VSC_SCRATCH_VO/$USER`)
* `$VSC_DATA_VO`
* `$VSC_DATA_VO_USER`